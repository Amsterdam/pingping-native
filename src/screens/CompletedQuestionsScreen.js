import React from 'react';
import {View, StyleSheet, Image} from 'react-native';
import {Content, Container, Left} from 'native-base';
import ContentLayout from '../components/layout/ContentLayout';
import Title from '../components/typography/Title';
import Body from '../components/typography/Body';
import Button from '../components/Button';
import HeaderTemplate from '../components/header/HeaderTemplate';
import HeaderBackButton from '../components/header/HeaderBackButton';
import ThumbsUp from '../assets/thumbs-up.png';

const CompletedQuestionsScreen = ({navigation, setLogin}) => {
  return (
    <Container>
      <HeaderTemplate style={styles.header} color="primary">
        <View>
          <Left style={styles.headerButtonContainer}>
            <HeaderBackButton navigation={navigation} />
          </Left>
        </View>
        <Title style={styles.title}>LEKKER</Title>
        <View style={styles.absolute}>
          <View style={styles.paper}>
            <Image source={ThumbsUp} />
          </View>
        </View>
      </HeaderTemplate>
      <ContentLayout style={styles.content}>
        <Content>
          <Body style={styles.paragraph1}>
            Met de antwoorden die jij hebt gegeven hebben wij voor jou een
            persoonlijke route samengesteld.
          </Body>
          <Body style={styles.paragraph2}>
            Bekijk wat jij moet doen om je basis op orde te hebben!
          </Body>
          <Button rounded label="Naar mijn Route" onPress={setLogin} />
        </Content>
      </ContentLayout>
    </Container>
  );
};

export default CompletedQuestionsScreen;

const styles = StyleSheet.create({
  header: {
    height: 200,
    flexDirection: 'column',
    justifyContent: 'flex-start',
    alignItems: 'flex-start',
  },
  headerButtonContainer: {
    flex: 0,
  },
  title: {
    color: '#fff',
    letterSpacing: 5,
    alignSelf: 'center',
  },
  paragraph1: {
    paddingBottom: 20,
  },
  paragraph2: {
    paddingBottom: 100,
  },
  button: {
    marginBottom: 15,
    marginTop: 15,
  },
  content: {
    marginTop: 150,
  },
  absolute: {
    position: 'absolute',
    top: 225,
    left: 0,
    right: 0,
    bottom: 0,
    justifyContent: 'center',
    alignItems: 'center',
  },
  paper: {
    backgroundColor: '#fff',
    padding: 100,
    height: 50,
    width: 50,
    shadowColor: '#000',
    shadowOffset: {
      width: 0,
      height: 2,
    },
    shadowOpacity: 0.25,
    shadowRadius: 3.84,
    elevation: 5,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
