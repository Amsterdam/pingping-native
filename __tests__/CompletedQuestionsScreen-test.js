import React from 'react';
import CompletedQuestionsScreen from '../src/screens/CompletedQuestionsScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<CompletedQuestionsScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
