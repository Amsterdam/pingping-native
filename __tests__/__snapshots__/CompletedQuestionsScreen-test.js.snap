// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "#0d2036",
        }
      }
    >
      <View
        noShadow={true}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "transparent",
              "borderBottomWidth": 0.5,
              "elevation": 0,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": 18,
              "right": 0,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
              "top": 0,
            },
            Object {
              "alignItems": "flex-start",
              "backgroundColor": "#0d2036",
              "flexDirection": "column",
              "height": 200,
              "justifyContent": "flex-start",
            },
          ]
        }
        transparent={true}
      >
        <View
          style={Object {}}
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "flex-start",
                  "alignSelf": "center",
                  "flex": 1,
                },
                Object {
                  "flex": 0,
                },
              ]
            }
          >
            <View
              accessible={true}
              focusable={true}
              forwardedRef={[Function]}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderBottomWidth": null,
                  "borderColor": "#007aff",
                  "borderLeftWidth": null,
                  "borderRadius": 5,
                  "borderRightWidth": null,
                  "borderTopWidth": null,
                  "borderWidth": undefined,
                  "elevation": 0,
                  "flexDirection": "row",
                  "height": 45,
                  "justifyContent": "space-between",
                  "opacity": 1,
                  "paddingBottom": 6,
                  "paddingTop": 6,
                  "shadowColor": null,
                  "shadowOffset": null,
                  "shadowOpacity": null,
                  "shadowRadius": null,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": undefined,
                      "fontSize": 12,
                    },
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 30,
                      },
                      Object {
                        "color": "#007aff",
                        "fontSize": 24,
                        "marginLeft": 16,
                        "marginRight": 16,
                        "paddingTop": 2,
                      },
                      Object {
                        "color": "#fff",
                      },
                    ],
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                type="AntDesign"
              >
                î˜œ
              </Text>
            </View>
          </View>
        </View>
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "alignSelf": "center",
                "color": "#fff",
                "fontFamily": "Heavitas",
                "fontSize": 30,
                "letterSpacing": 5,
                "textAlign": "left",
              },
            ]
          }
          uppercase={false}
        >
          LEKKER
        </Text>
        <View
          style={
            Array [
              Object {},
              Object {
                "alignItems": "center",
                "bottom": 0,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 225,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "elevation": 5,
                  "height": 50,
                  "justifyContent": "center",
                  "padding": 100,
                  "shadowColor": "#000",
                  "shadowOffset": Object {
                    "height": 2,
                    "width": 0,
                  },
                  "shadowOpacity": 0.25,
                  "shadowRadius": 3.84,
                  "width": 50,
                },
              ]
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/thumbs-up.png",
                }
              }
            />
          </View>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "marginTop": 150,
        "padding": 30,
        "position": "relative",
        "width": 750,
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
    >
      <RCTScrollView
        automaticallyAdjustContentInsets={false}
        contentContainerStyle={
          Array [
            Object {
              "padding": undefined,
            },
            undefined,
          ]
        }
        contentInset={
          Object {
            "bottom": 0,
          }
        }
        enableAutomaticScroll={true}
        enableOnAndroid={false}
        enableResetScrollToCoords={true}
        extraHeight={75}
        extraScrollHeight={0}
        getScrollResponder={[Function]}
        handleOnScroll={[Function]}
        keyboardDismissMode="interactive"
        keyboardOpeningTime={250}
        keyboardShouldPersistTaps="handled"
        keyboardSpace={0}
        onScroll={[Function]}
        resetKeyboardSpace={[Function]}
        resetScrollToCoords={
          Object {
            "x": 0,
            "y": 0,
          }
        }
        scrollEventThrottle={1}
        scrollForExtraHeightOnAndroid={[Function]}
        scrollIntoView={[Function]}
        scrollToEnd={[Function]}
        scrollToFocusedInput={[Function]}
        scrollToPosition={[Function]}
        showsVerticalScrollIndicator={true}
        style={
          Object {
            "backgroundColor": "transparent",
            "flex": 1,
          }
        }
        update={[Function]}
        viewIsInsideTabBar={false}
      >
        <View>
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Raleway",
                  "fontSize": 15,
                  "lineHeight": 25,
                  "paddingBottom": 20,
                  "textAlign": "left",
                },
              ]
            }
            uppercase={false}
          >
            Met de antwoorden die jij hebt gegeven hebben wij voor jou een persoonlijke route samengesteld.
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Raleway",
                  "fontSize": 15,
                  "lineHeight": 25,
                  "paddingBottom": 100,
                  "textAlign": "left",
                },
              ]
            }
            uppercase={false}
          >
            Bekijk wat jij moet doen om je basis op orde te hebben!
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#fb9f00",
                "borderBottomWidth": null,
                "borderColor": false,
                "borderLeftWidth": null,
                "borderRadius": 57,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": undefined,
                "elevation": 5,
                "flexDirection": "row",
                "height": 45,
                "justifyContent": "center",
                "marginBottom": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.25,
                "shadowRadius": 3.84,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#fff",
                    "fontFamily": "System",
                    "fontSize": 16.5,
                    "marginLeft": 0,
                    "marginRight": 0,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                  },
                  Object {
                    "color": "#fff",
                    "fontFamily": "Heavitas",
                    "fontSize": 14,
                  },
                ]
              }
              uppercase={false}
            >
              Naar mijn Route
            </Text>
          </View>
        </View>
      </RCTScrollView>
    </RCTSafeAreaView>
  </View>
</View>
`;
